### PR TITLE
Added OnDA Monitor recipe

### PR DESCRIPTION
**Pull request by valmar**
Adding OnDA Monitor (OM)

### CODE REVIEW

File: recipes/ondamonitor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  skip: true  # [not linux64]
+
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - python
+    - make
+    - setuptools
+    - pip
```
Comment (dopplershift) [inline recipes/ondamonitor/meta.yaml:24]:
```suggestion
    - make
```

File: recipes/ondamonitor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  host:
+    - python
+    - numpy
```
Comment (dopplershift) [inline recipes/ondamonitor/meta.yaml:27]: You can put `setuptools` here.

File: recipes/ondamonitor/meta.yaml (hunk 1 above)
Comment (valmar) [reply recipes/ondamonitor/meta.yaml:24]: Thank you

File: recipes/ondamonitor/meta.yaml (hunk 2 above)
Comment (valmar) [reply recipes/ondamonitor/meta.yaml:24]: Done, thank you!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**